tst/EnergyPlus/unit/HVACFan.unit.cc

```diff
@@ -133,7 +133,7 @@ TEST_F(EnergyPlusFixture, SystemFanObj_FanSizing1)
     });
 
     ASSERT_TRUE(process_idf(idf_objects));
-
+    DataEnvironment::StdRhoAir = 1.0;
```

Comment:
Set `StdRhoAir` to something other than zero, here and elsewhere.

Reply:
I've actually thought about adding some code to the `EnergyPlusFixture::setUp()` function that default-initializes some of these things.  In a separate issue, I found the time step values that needed to be initialized in order to get the schedule manager working.  And it seems they are initialized in a ton of unit tests.  We might as well set up a meaningful default value for our unit testing and then let individual unit tests override it to do specific testing if necessary.  I'm not saying that needs to be done here, but maybe this is another thing that should be on the list of stuff to default-initialize.